tests/MongoDB.EntityFrameworkCore.FunctionalTests/Query/OwnedEntityTests.cs

```diff
@@ -96,12 +96,14 @@ public void OwnedEntity_nested_one_level_creates()
         }
     }
 
-    [Fact(Skip = "We do not yet support updating owned types")]
-    public void OwnedEntity_nested_collection_creates()
+    [Fact]
```

Comment:
We probably need another test for the case when an entity has 2 navigation properties of same type. Like: 
```
class Person
{
  public Location PrimaryLocation { get; set; }
  public Location SecondaryLocation { get; set; }
}
```

Reply:
Can't add this one yet as it would fail - it's a known bug EF-58.